portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -29,7 +29,10 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** This servlet handles comment's data: 
+ - doPost() stores get's the recently submitted comment's data from the request 
+            and stores in a database using Datastore API
+ - doGet() returns the comments from the database, aftern converting them to JSON*/
```

Comment:
If the Javadoc has multiple lines, we should include the `*` on all new lines. 

Example:
```
/**
 * Here is my first line.
 * The second line.
 */
```

https://google.github.io/styleguide/javaguide.html#s7.1-javadoc-formatting

Reply:
done